src/components/EditUser.vue

```diff
@@ -14,7 +14,7 @@
             v-validate="'min:8|max:80'" />
         </b-field>
         <b-field label="Updated first name" :type="{'is-danger': errors.has('name')}" :message="errors.first('name')">
-          <b-input placeholder="Updated first name" v-model="new_name" name="name" v-validate="'required|alpha|max:80'" />
```

Comment:
To chyba sprawi że będziemy mogli wpisać cyferki w imię, lepiej zrobić alpha_spaces